src/imitation/scripts/convert_trajs_in_place.py

```diff
@@ -0,0 +1,42 @@
+"""
+This script takes as command-line input multiple paths to pickle files containing
+Sequence[imitation.types.TrajectoryWithRew] or Sequence[imitation.old_types.Trajectory].
+It overwrites each path with a new pickle where the demonstration data is saved as
+Sequence[TrajectoryWithRew].
```

Comment:
Is this an `imitation.types.TrajectoryWithRew`? Docstring currently makes it sound like it takes types A or B and converts them to C, but I think it's actually A or B and converts them to A?

Reply:
That makes sense -- expanding this reference